chrome/locale/en-US/zotero/zotero.properties

```diff
@@ -968,3 +968,15 @@ firstRunGuidance.quickFormat	= Type a title or author to search for a reference.
 firstRunGuidance.quickFormatMac	= Type a title or author to search for a reference.\n\nAfter you've made your selection, click the bubble or press Cmd-\u2193 to add page numbers, prefixes, or suffixes. You can also include a page number along with your search terms to add it directly.\n\nYou can edit citations directly in the word processor document.
 firstRunGuidance.toolbarButton.new = Click here to open Zotero, or use the %S keyboard shortcut.
 firstRunGuidance.toolbarButton.upgrade = The Zotero icon can now be found in the Firefox toolbar. Click the icon to open Zotero, or use the %S keyboard shortcut.
+
+cslpreview.instruction		= Select one or more items in Zotero and click the "Refresh" button to see how these items are rendered by the installed CSL citation styles.
+cslpreview.warning.noItems		= No items selected in Zotero.
+
+csledit.saveAs.title		= Save Citation Style
+csledit.saveAs.CSL		= Citation Style Language
```

Comment:
Let's pick some prefix for bib-related stuff and put this and "Bibliography" and other stuff there. Maybe ```citations.citationStyleLanguage```, ```citations.bibliography```, etc.?